### ISSUE TITLE
Make an Appointment > Content

### ISSUE THREAD

**Issue by brentrobbins** [variantstudios/wisdomwaysacupuncture.com#15]
Content (text, links & images):
- [ ] Make an Appointment
- [ ] Forms To Complete
- [ ] Payment Info

### Notes
You wanted me to send you a note to remind you of that last thing we talked about last night, in regards to my “schedule an appt” and “forms to complete” pages. 

Basically, the gist of it is that for the 3 types of new patients, 
All New Patients
Fertility 
Insurance,

I need to make sure they get shunted through the process for doing everything they need to to schedule correctly and do what they need to do before coming in. 

That mostly means printing and completing the correct forms related to their type, which brings us to the “forms to complete” page. All the relevant forms are currently listed and linked to under the correct category of patient type. I think it’s pretty clear on my current website, but people still either miss some steps of the process or bring the wrong forms. And it just looks ugly the way it is now as just lists of instructions and links going down the page. 

But how it goes is, everyone need to fill out the basic forms related to new patients.  
Then, fertility patients have an additional form they need to complete, depending on if they’re male or female. 
In addition, if any of these patients want to be able to have us bill insurance at their first visit, they also need to have filled out the insurance verification form.

I’m thinking what might be good is to have 3 large “buttons” or boxes of some kind, for the 3 types of patients, each containing the links to the relevant forms. Or any other idea you think looks good and is user friendly/intuitive.

A common current issue is that even though I think it’s pretty darned obvious on my current website, we constantly have people coming in wanting to bill insurance without having filled out and verified their insurance. And they’ll sometimes act oblivious like they didn’t realize that they needed to have that form filled out (which I know really means they just never got around to calling their insurance since it’s a PITA, because I can’t see how they can possibly not see the instructions as they’re so clear). Anyway, if you have an idea for how we can make it absolutely unmistakable what steps they need to do if they want to bill insurance, that would be good. 

I’m not sure if it might be a good idea to have a box right at the top related to insurance patients, so everyone sees that first? I dunno, I’m kinda lost so any idea you have.

Another issue we sometimes have is that male fertility patients will print out BOTH the male AND female forms, or vise versa. Again, I think it’s pretty obvious that they don’t need to do that, so probably people just aren’t looking closely at all, but if you can think of a way to maybe like, make it a button they click or something where ONLY their relevant forms appear, that could be good. Or any other handy idea you have. 

Also, once this new setup is in place, do you think we’ll even NEED 3 separate pages for Schedule Appt, Forms, Fees?  If it can be arranged well, it’s kinda seeming like they might all be to be condensed onto one page?
